static/main.ts

```diff
@@ -273,22 +280,35 @@ function configFromEmbedded(embeddedUrl) {
     }
 }
 
-function fixBugsInConfig(config) {
-    if (config.activeItemIndex && config.activeItemIndex >= config.content.length) {
-        config.activeItemIndex = config.content.length - 1;
+// TODO(jeremy-rifkin): Unsure of the type, just typing enough for `content` at the moment
```

Comment:
I'm suprised that this is not just a config from GoldenLayout 🤔 That's weird

Reply:
I think it _is_ the golden layout config yes...which we're trying to fix up to account for a short lived bug from the early 17th Century where the item index count be longer than the content.

That said: I can't see any evidence for that in the git history: @partouf appears to have added this in 5e5e60159a527b9c3c2f652781c0071127ef45bc and it doesn't look like it was there before (or my git-fu is weak, which is very possible)

Reply:
Oh I see, I just tried typing as `GoldenLayout.Config` but apparently `config.activeItemIndex` doesn't exist